### ISSUE TITLE
Graph Overview red rectangle wrong position

### ISSUE THREAD

**Issue by idandre** [radareorg/cutter#1536]
**Environment information**
* Operating System: arch 
* Cutter version: 1.8.1
* File format: elf64
 * Arch: x86
 * Type: DYN (Shared object file)

TL;DR_
Red rectangle in Graph Overview is not were it is supposed to be after entering a Comment. 

**Describe the bug**
After todays( may 15) update to 1.8.1 i added a comment by pushing [;]. After finishing and pressing [ENTER] the red rectangle in Graph Overview was not were it is supposed to be. It moved nearly on screen width to the left. The red rectangle is still attached to the mouse, when I hold left mouse button. The red rectangle is positioned 1 screen to the left. After restart everything was fine, as long as i did not finish a comment. After the third restart of cutter trying to reproduce the behavior, fixed it in the end ... No after the 4th restart it occured again:)

**To Reproduce**
1. push [;]
2. enter comment
3. push [ENTER]
4. click left mouse button and move graph

**Expected behavior**
Graph Overview should show a red rectangle accordingly to shown section in the main graph.

**Screenshots**
![Bildschirmfoto_2019-05-15_14-17-40](https://user-images.githubusercontent.com/41512496/57775016-54fa7300-771c-11e9-8221-0421604b8f4b.png)

**Additional context**

**Comment by karliss** [comment]
Could it be that red rectangle is in correct place but the image isn't updated? I wasn't repeat what you described, but I observed that after adding large comment look image in the overview isn't updated. Changes in graph could cause differences in basic block placement. If the overview is showing outdated cached image it could like the position of red rectangle is wrong.

**Comment by idandre** [comment]
Hi karliss,

Thanks for your fast reply. This could be possible. Because it matches at the bottom of the page but not at the top. I added a comment at a block in the center and some photos showing some more variant of the same function.

By the way. There should be a possiblity to switch off auto adjusting after pushing [j],[k] to go up and down in a block. Because if the code block is to big you it moves to the top but you would like to work at the bottom and cant see anything in graph view 😋

Good Night.
________________________________
Von: karliss <notifications@github.com>
Gesendet: Mittwoch, 15. Mai 2019 21:19
An: radareorg/cutter
Cc: ReverseiT; Author
Betreff: Re: [radareorg/cutter] Graph Overview red rectangle wrong position (#1536)


Could it be that red rectangle is in correct place but the image isn't updated? I wasn't repeat what you described, but I observed that after adding large comment look image in the overview isn't updated. Changes in graph could cause differences in basic block placement. If the overview is showing outdated cached image it could like the position of red rectangle is wrong.
